tests/single/rpc/test_eth_getUserOperationByHash.py

```diff
@@ -22,9 +22,8 @@ def test_eth_getUserOperationByHash(helper_contract, userop, schema):
     Validator.check_schema(schema)
     validate(instance=response.result, schema=schema)
 
-
 def test_eth_getUserOperationByHash_error():
     response = RPCRequest(method="eth_getUserOperationByHash", params=[""]).send()
     assert_rpc_error(
-        response, "Missing/invalid userOpHash", RPCErrorCode.INVALID_FIELDS
+        response, "", RPCErrorCode.INVALID_FIELDS
```

Comment:
wouldnt this just check the response is an empty string?

Reply:
that's a good point. I am going to change this to ignore the error message.